src/ifd-nfc.h

```diff
@@ -31,4 +31,8 @@
 #define IFDNFC_SET_ACTIVE_SE     2
 #define IFDNFC_GET_STATUS        3
 
+#ifndef IFD_NFC_OPEN_RETRY_INTERVAL 
```

Comment:
Thanks.  I did it this way so that if someone was not happy with this default they could override it in compiler flags.